niceman/support/distributions/tests/test_debian.py

```diff
@@ -285,6 +285,10 @@ def test_parse_apt_cache_policy_source_info():
  500 http://security.ubuntu.com/ubuntu xenial-security/restricted amd64 Packages
      release v=16.04,o=Ubuntu,a=xenial-security,n=xenial,l=Ubuntu,c=restricted,b=amd64
      origin security.ubuntu.com
+ 500 http://debproxy:9999/debian/ jessie-backports/contrib Translation-en
+ 100 http://debproxy:9999/debian/ jessie-backports/non-free amd64 Packages
+     release o=Debian Backports,a=jessie-backports,n=jessie-backports,l=Debian Backports,c=non-free
+     origin debproxy
```

Comment:
I believe I added it on the commit but it doesn't show here -- could we have a test which tests that value is parsed correctly?

Reply:
That is a quote right from your bug report, but I cut out the exact portion that caused the problem.  I'm happy to add more if you wish...

Reply:
What I meant -- could the test verify correct parsing of this line -- that we are getting correct record with `origin="Debian Backports"` etc?

PS as for what "doesn't show here" -- it was about my comment, not the code ;)

Reply:
Done ( 496efe8 )